org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/core/JDTCompilerAdapter.java

```diff
@@ -245,8 +245,8 @@ protected Commandline setupJavacCommand() throws BuildException {
 		if (this.attributes.getNowarn()) {
 			// disable all warnings
 			Object[] entries = this.customDefaultOptions.entrySet().toArray();
-			for (int i = 0, max = entries.length; i < max; i++) {
-				Map.Entry entry = (Map.Entry) entries[i];
+			for (Object entry2 : entries) {
```

Comment:
Shouldn't this iterate directly over set, without extra array?

Reply:
yea that would be better, i have also seen other odds. For example ParameterizedQualifiedTypeReference needs the refactoring twice.
Should i squash manual changes or add another commit?

Reply:
I would expect extra commit for all non-automated changes.